### PR TITLE
Migrated RM114 to new validation

### PR DESCRIPTION
**Pull request by JasBeilin**
## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-9807

## Description
Migrated the validation to get images path and validate it exists.

### CODE REVIEW

File: demisto_sdk/commands/validate/validators/RM_validators/RM114_is_image_exists_in_readme.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+
+class IsImageExistsInReadmeValidator(BaseValidator[ContentTypes]):
+    error_code = "RM114"
+    description = "Validate README images are actually exits."
+    error_message = "The following image files does not exists: {0}"
+    related_field = ""
+    is_auto_fixable = False
+    related_file_type = [RelatedFileType.README]
+    # expected_git_statuses = [GitStatuses.MODIFIED, GitStatuses.ADDED]
```
Comment (YuvHayun) [inline demisto_sdk/commands/validate/validators/RM_validators/RM114_is_image_exists_in_readme.py:25]: remove

File: demisto_sdk/commands/validate/validators/RM_validators/RM114_is_image_exists_in_readme.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            if (
+                any(
+                    invalid_lines := [
+                        f"Packs/{content_item.pack_name}/{str(image_path).replace('../', '')}"
+                        if content_item.pack_name not in str(image_path)
+                        else str(image_path)
+                        for image_path in extract_image_paths_from_str(
+                            text=content_item.readme
+                        )
+                        if image_path and not image_path.is_file()
```
Comment (YuvHayun) [inline demisto_sdk/commands/validate/validators/RM_validators/RM114_is_image_exists_in_readme.py:44]: isn't it a bit different from the original validation?
Looks like the original one alter the path and only then check if the file exist.

File: demisto_sdk/commands/validate/tests/test_tools.py [hunk 3]
```diff
@@ -93,6 +101,8 @@ def create_correlation_rule_object(
 def create_playbook_object(
     paths: Optional[List[str]] = None,
     values: Optional[List[Any]] = None,
+    pack_info: Optional[Dict[str, Any]] = None,
+    readme_content: Optional[str] = None,
```
Comment (YuvHayun) [inline demisto_sdk/commands/validate/tests/test_tools.py:105]: Please add to doc strings.

**Review by YuvHayun — APPROVED** [review]
Nice!
Minor comments.

File: demisto_sdk/commands/validate/validators/RM_validators/RM114_is_image_exists_in_readme.py (hunk 2 above)
Comment (JasBeilin) [reply demisto_sdk/commands/validate/validators/RM_validators/RM114_is_image_exists_in_readme.py:46]: thanks for catching it.

**Pull request merged**